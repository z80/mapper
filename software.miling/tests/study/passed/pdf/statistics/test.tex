
\documentclass{sb_cv}
 
\makeatletter
\newcommand*{\skypesocialsymbol}{\includegraphics[height=.7\baselineskip]{skype_gray}}
\collectionadd[skype]{socials}{ sergey\char`_831 }
\makeatother


\newcommand*{\skypesymbol} {%
  \protect\raisebox{-0.085em}{%
\protect\begin{tikzpicture}[y=0.08em,x=0.08em,xscale=0.022,yscale=-0.022, inner sep=0pt, outer sep=0pt]
\protect\path[fill=color2,even odd rule] (487.6550,288.9690) .. controls (489.0610,278.5690) and
  (489.8700,267.9960) .. (489.8700,257.2330) .. controls (489.8700,128.0770) and
  (384.5990,23.3610) .. (254.7670,23.3610) .. controls (241.8630,23.3610) and
  (229.2120,24.4210) .. (216.9010,26.4410) .. controls (194.8280,12.0570) and
  (168.5590,3.6740) .. (140.2880,3.6740) .. controls (62.7660,3.6740) and
  (0.0000,66.4820) .. (0.0000,143.9800) .. controls (0.0000,172.1780) and
  (8.2990,198.3740) .. (22.5900,220.3690) .. controls (20.6650,232.3860) and
  (19.6810,244.6920) .. (19.6810,257.2290) .. controls (19.6810,386.4050) and
  (124.8980,491.1100) .. (254.7660,491.1100) .. controls (269.4230,491.1100) and
  (283.6930,489.6840) .. (297.5620,487.1780) .. controls (319.1120,500.5470) and
  (344.4960,508.3260) .. (371.7080,508.3260) .. controls (449.2100,508.3260) and
  (512.0010,445.5020) .. (512.0010,368.0120) .. controls (511.9980,338.7190) and
  (503.0410,311.4840) .. (487.6550,288.9690) -- cycle(276.7400,429.5960) ..
  controls (202.0340,433.4870) and (167.0750,416.9590) .. (135.0500,386.9050) ..
  controls (99.2850,353.3370) and (113.6520,315.0500) .. (142.7900,313.1040) ..
  controls (171.9120,311.1590) and (189.3980,346.1160) .. (204.9410,355.8400) ..
  controls (220.4650,365.5280) and (279.5340,387.6000) .. (310.7350,351.9320) ..
  controls (344.7100,313.1040) and (288.1410,293.0120) .. (246.6760,286.9300) ..
  controls (187.4730,278.1640) and (112.7260,246.1370) .. (118.5410,183.0230) ..
  controls (124.3580,119.9490) and (172.1230,87.6090) .. (222.3910,83.0470) ..
  controls (286.4680,77.2300) and (328.1820,92.7540) .. (361.1760,120.9070) ..
  controls (399.3270,153.4360) and (378.6840,189.8010) .. (354.3770,192.7270) ..
  controls (330.1660,195.6360) and (302.9730,139.2230) .. (249.5860,138.3750) ..
  controls (194.5590,137.5110) and (157.3690,195.6360) .. (225.3000,212.1590) ..
  controls (293.2660,228.6640) and (366.0500,235.4450) .. (392.2610,297.5760) ..
  controls (418.4900,359.7130) and (351.5070,425.7010) .. (276.7400,429.5960) --
  cycle;
\protect\end{tikzpicture}}%
  ~}

\newcommand*{\skype}[1]{\def\@skype{#1}}


% personal data
\name{Sergey}{Bashkirov}
%\title{Resum√© title}                               % optional, remove / comment the line if not wanted
\address{1020 3rd street}{94945 Novato California}{USA}% optional, remove / comment the line if not wanted; the "postcode city" and and "country" arguments can be omitted or provided empty
\email{bashkirov.sergey@gmail.com}                               % optional, remove / comment the line if not wanted
\phone[mobile]{+1~(415)~246~0343}                   % optional, remove / comment the line if not wanted
\break
%\homepage{ajnorton.com}                         % optional, remove / comment the line if not wanted
\social[github][github.com/z80]{github.com/z80}
\social[linkedin][https://www.linkedin.com/in/sergey-bashkirov]{linkedin.com/in/sergey-bashkirov}
%\extrainfo{additional information}                 % optional, remove / comment the line if not wanted
%photo[64pt][0.4pt]{picture}                       % optional, remove / comment the line if not wanted; '64pt' is the height the picture must be resized to, 0.4pt is the thickness of the frame around it (put it to 0pt for no frame) and 'picture' is the name of the picture file


%\quote{
%Software Engineer with significant R\&D experience in Atomic Force Microscopy, Raman TERS imaging with a focus on software and firmware algorithms development. Active opensource community participant. I'm seeking for a full time position in software/firmware development.
%}

% to show numerical labels in the bibliography (default is to show no labels); only useful if you make citations in your resume
%\makeatletter
%\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}
%\makeatother
%\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}% CONSIDER REPLACING THE ABOVE BY THIS

% bibliography with mutiple entries
%\usepackage{multibib}
%\newcites{book,misc}{{Books},{Others}}
%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
%\begin{CJK*}{UTF8}{gbsn}                          % to typeset your resume in Chinese using CJK
%-----       resume       ---------------------------------------------------------
\makecvtitle
\vspace*{-15mm}

\section{Summary}
Application software engineer with R\&D experience in software development for scientific devices with active open source community participation; experienced in GUI design, machine learning applications, firmware design; have experience in algorithms development, data acquisition and processing, computer vision; good develpment time management abilities, problem solving R\&D experience.

\vspace*{-4mm}
\section{Technical proficiencies}
\cvdoubleitem{Programming}{C, C++, Java}{Scripting}{Lua, Ruby, Python, R, SQL, Octave, Matlab, Shell scripting}
\cvdoubleitem{Platforms}{Linux, Windows}{Web technologies}{Ruby-on-rails4, Javascript, Bootstrap}
\cvdoubleitem{Frameworks}{Qt, Boost, Swing, WxWidgets}{Math applications}{Data acquisition, data processing, filtering, automation, regression model extraction, Kalman filtering}

\vspace*{-4mm}
\section{Personal achievements}
\begin{itemize}

\item Achieved high precision with equipment made of low cost components by applying machine learning techniques to calibration process.

\item Reorganaized production and supplies purchase processes by perforing statistical predictive contracts analysis. Made business control software for warehouse keeping, product assembling, purchasing and contracts tracking.

\item Simplified "find the same place" task for AFM after sample reinstallation by making video aligning and navigation by video mode.

\item Created from scratch software for controlling Atomic Force Microscope.

\item Reduced AFM modes development time and eased prototyping by implementing scriptable GUI builder and hardware embedded real time scripting language.
\end{itemize}


\vspace*{-4mm}
\section{Most recent work experience}
\cventry{}{Software Engineer}{Aist-NT Inc.}{Novato, CA}{}{AFM software development, firmware development, image processing, AFM modes design, AFM-Raman coupling, actuators control, data acquisition and processing, production control}  % arguments 3 to 6 can be left empty



\cventry{}{Contractor}{Transmag}{Santa Rosa, CA}{}{USB based BLDC motor controller's interface development. Designed firmware, user interface, suggested proper USB schematics.}

\vspace*{-4mm}
\section{Education}
\cventry{}{Master of Science in Applied Mathematics and Physics}{Moscow Institute of Physics and Technology}{Moscow, Russia}{}{}
\begin{itemize}
\item \textbf{Most recent courses taken:} Circuits \& Electronics, Machine learning, SAAS, Kalman filtering, Autonomous navigation, Statistics with R, Node.JS, Angular.JS, Booststrap, JavaScript.
\end{itemize}


\vspace*{-4mm}
\section{Open source projects participation}
\cvitemwithcomment{grambo-pi.com}{Created expandable stackable PCBs set for RaspberryPi computer for robot prototyping.}{}
\cvitemwithcomment{xonotic.org}{Created Blend2map Blender3D to MAP exporter.}{}
\cvitemwithcomment{chibios.org}{Submitted I2C slave mode driver, IWDG driver.}{}
\cvitemwithcomment{QtLua}{Provided a patch making Object::connect() work in the same way as in Qt.}{}
\cvitemwithcomment{Gaw}{Provided a patch fixing crashes with default configuration file for gaw waveform viewer.}{}



\end{document}