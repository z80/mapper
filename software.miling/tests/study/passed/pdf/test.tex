
\documentclass{sb_cv}
 
% adjust the page margins
%\setlength{\hintscolumnwidth}{3cm}                % if you want to change the width of the column with the dates
%\setlength{\makecvtitlenamewidth}{10cm}           % for the 'classic' style, if you want to force the width allocated to your name and avoid line breaks. be careful though, the length is normally calculated to avoid any overlap with your personal info; use this at your own typographical risks...

% personal data
\name{Sergey}{Bashkirov}
%\title{Resum√© title}                               % optional, remove / comment the line if not wanted
\address{1020 3rd street}{94945 Novato California}{USA}% optional, remove / comment the line if not wanted; the "postcode city" and and "country" arguments can be omitted or provided empty
\email{litedictteam@gmail.com}                               % optional, remove / comment the line if not wanted
\phone[mobile]{+1~(415)~123~4567}                   % optional, remove / comment the line if not wanted
\break
%\homepage{ajnorton.com}                         % optional, remove / comment the line if not wanted
\social[github][github.com/z80]{github.com/z80}
\social[linkedin][https://www.linkedin.com/in/sergey-bashkirov]{linkedin.com/in/sergey-bashkirov}
%\extrainfo{additional information}                 % optional, remove / comment the line if not wanted
%photo[64pt][0.4pt]{picture}                       % optional, remove / comment the line if not wanted; '64pt' is the height the picture must be resized to, 0.4pt is the thickness of the frame around it (put it to 0pt for no frame) and 'picture' is the name of the picture file


%\quote{
%Software Engineer with significant R\&D experience in Atomic Force Microscopy, Raman TERS imaging with a focus on software and firmware algorithms development. Active opensource community participant. I'm seeking for a full time position in software/firmware development.
%}

% to show numerical labels in the bibliography (default is to show no labels); only useful if you make citations in your resume
%\makeatletter
%\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}
%\makeatother
%\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}% CONSIDER REPLACING THE ABOVE BY THIS

% bibliography with mutiple entries
%\usepackage{multibib}
%\newcites{book,misc}{{Books},{Others}}
%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
%\begin{CJK*}{UTF8}{gbsn}                          % to typeset your resume in Chinese using CJK
%-----       resume       ---------------------------------------------------------
\makecvtitle

\section{Goal}
Software Engineer with R\&D experience in software and firmware development related to Atomic Force Microscopy, Raman TERS imaging with active open source community participation is seeking for a full time position in software/firmware development.

\section{Areas of expertise}
\cvdoubleitem{Crossplatform programming}{GUI, hardware communication}{RPC design}{Sockets, XMPP, ICE, XMLRPC}
\cvdoubleitem{Firmware design}{Real time firmware, actuators control, physical properties measurement, RTOS and bare metal firmware}{Scripting and automation}{Scripting languages, embedded scripting, bare metal firmware scripting}
\cvdoubleitem{Visualization}{Widgets design, Spatial, 3D geometry, 2D/3D visualization}{Math}{Computer vision, Kalman filtering, machine learning, statistics, data, image processing }

\section{Technical proficiencies}
\cvdoubleitem{Programming}{C, C++, Java, microcontroller assembler}{Scripting}{Lua, Ruby, Python, R, SQL, Shell scripting}
\cvdoubleitem{Platforms}{Linux, Windows, ChibiOs, FreeRTOS}{Web technologies}{Ruby-on-rails4, Javascript, Bootstrap}
\cvdoubleitem{Frameworks}{Qt, Boost, Swing, WxWidgets}{Various}{FTDI/CPLD Verilog programming, simulation, PCB design, Spice simulation, hardware debugging, mature soldering skill}

\section{Most recent work experience}
\cventry{}{Software Engineer}{Aist-NT Inc.}{Novato, CA}{}{}  % arguments 3 to 6 can be left empty
\begin{itemize}
\item Created and maintained main AFM software.
\item Designed scriptable GUI building framework and embedded real time scripting language.
\item Created firmware for a number of devices, designed PCB for a few hardware solutions.
\item Created WEB based software for warehouse keeping, production control, expenses calculation, forecasting.
\end{itemize}


\cventry{}{Contractor}{Transmag}{Santa Rosa, CA}{}{USB based BLDC motor controller development.}
\begin{itemize}
\item Designed GUI for BLDC motor setup and testing.
\item Designed firmware for USB interaction and BLDC parameters read/write.
\item Suggested USB part schematic design in order to provide reliabe USB communication.
\end{itemize}


\section{Education}
\cventry{}{Master of Science in Applied Mathematics and Physics}{Moscow Institute of Physics and Technology}{Moscow, Russia}{}{}
\begin{itemize}
\item Circuits \& Electronics, Machine learning, SAAS, Autonomous navigation, Statistics with R, Node.JS, Angular.JS, Booststrap, JavaScript.
\end{itemize}

\section{Open source projects participation}
\cvitemwithcomment{grambo-pi.com}{Created expandable stackable PCBs set for RaspberryPi computer for robot prototyping.}{}
\cvitemwithcomment{xonotic.org}{Created Blen2map Blender3D to MAP exporter.}{}
\cvitemwithcomment{chibios.org}{Submitted I2C slave mode driver, IWDG driver.}{}
\cvitemwithcomment{QtLua}{Provided a patch making Object::connect() work in the same way as in Qt.}{}
\cvitemwithcomment{Gaw}{Provided a patch fixing crashes with default configuration file for gaw waveform viewer.}{}



\end{document}