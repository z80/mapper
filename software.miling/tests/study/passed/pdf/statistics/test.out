\BOOKMARK [1][]{section*.1}{\376\377\000S\000u\000m\000m\000a\000r\000y}{}% 1
\BOOKMARK [1][]{section*.2}{\376\377\000T\000e\000c\000h\000n\000i\000c\000a\000l\000\040\000p\000r\000o\000f\000i\000c\000i\000e\000n\000c\000i\000e\000s}{}% 2
\BOOKMARK [1][]{section*.3}{\376\377\000P\000e\000r\000s\000o\000n\000a\000l\000\040\000a\000c\000h\000i\000e\000v\000e\000m\000e\000n\000t\000s}{}% 3
\BOOKMARK [1][]{section*.4}{\376\377\000M\000o\000s\000t\000\040\000r\000e\000c\000e\000n\000t\000\040\000w\000o\000r\000k\000\040\000e\000x\000p\000e\000r\000i\000e\000n\000c\000e}{}% 4
\BOOKMARK [1][]{section*.5}{\376\377\000E\000d\000u\000c\000a\000t\000i\000o\000n}{}% 5
\BOOKMARK [1][]{section*.6}{\376\377\000O\000p\000e\000n\000\040\000s\000o\000u\000r\000c\000e\000\040\000p\000r\000o\000j\000e\000c\000t\000s\000\040\000p\000a\000r\000t\000i\000c\000i\000p\000a\000t\000i\000o\000n}{}% 6
